src/Uno.UI.RuntimeTests/Tests/Windows_UI_Xaml_Controls/Given_CalendarDatePicker.cs

```diff
@@ -0,0 +1,34 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Windows.UI.Xaml.Controls;
+using Windows.UI.Xaml.Controls.Primitives;
+using static Private.Infrastructure.TestServices;
+
+namespace Uno.UI.RuntimeTests.Tests.Windows_UI_Xaml_Controls;
+
+[TestClass]
+[RunsOnUIThread]
+public class Given_CalendarDatePicker
+{
+	[TestMethod]
+	public async Task TestCalendarPanelSize()
+	{
+		var SUT = new CalendarDatePicker();
+		WindowHelper.WindowContent = SUT;
+		await WindowHelper.WaitForIdle();
+
+		var root = (Grid)SUT.FindName("Root");
+		var flyout = (Flyout)FlyoutBase.GetAttachedFlyout(root);
+		flyout.Open();
+
+		await WindowHelper.WaitForIdle();
+		var calendarView = (CalendarView)flyout.Content;
+
+		Assert.AreEqual(352, calendarView.ActualHeight);
```

Comment:
I wouldn't test equality he, as the templates may change slightly in the future and that would immediately break the test. Rather test something like height > 300 or similar. When the bug was present, the calendar was arranged 80 pixels high, so that should be quite safe buffer.